blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/util/GlobMatcher.java

```diff
@@ -0,0 +1,33 @@
+package io.jenkins.blueocean.service.embedded.util;
+
+import org.apache.oro.text.GlobCompiler;
+import org.apache.oro.text.regex.MalformedPatternException;
+import org.apache.oro.text.regex.Pattern;
+import org.apache.oro.text.regex.Perl5Matcher;
+
+/**
+ * Matches glob string patters
+ * See https://en.wikipedia.org/wiki/Glob_(programming)
+ */
+public final class GlobMatcher {
```

Comment:
This being there in `dependencyManagement` won't make other plugins depend on it. Only if those plugin define dependencies on it. I think any module depends on a maven library should be kept in dependencyManagement, enforces one version. Just like how we have workflow libraries in dependencyManagement but only module needing them are `blueocean-pipeline-api-impl`. I think its better we are consistent than make such exceptions.